scripts/download_artifacts.py

```diff
@@ -0,0 +1,140 @@
+import argparse
+import requests
+import os
+import importlib
+
+from github import Auth, Github, Repository
+from zipfile import ZipFile
+get_ci_hash = importlib.import_module("get-most-recent-ci-hash")
```

Comment:
Is there a way to do a regular or relative import rather than importlib?

Reply:
The only way to do so is to rename the scripts to remove the dashes in the filename

Reply:
We should do that then. Renaming isn't a big price to pay for something like this :)